### PR TITLE
WatchGuard.MobileVPNwithSSLclient version 12.7.2

### PR DESCRIPTION
**Pull request by itmunky**
- [ ] Have you signed the [Contributor License Agreement](https://cla.opensource.microsoft.com/microsoft/winget-pkgs)?
- [ ] Have you checked that there aren't other open [pull requests](https://github.com/microsoft/winget-pkgs/pulls) for the same manifest update/change?
- [ ] Have you [validated](https://github.com/microsoft/winget-pkgs/blob/master/AUTHORING_MANIFESTS.md#validation) your manifest locally with `winget validate --manifest <path>`?
- [ ] Have you tested your manifest locally with `winget install --manifest <path>`?
- [ ] Does your manifest conform to the [1.2 schema](https://github.com/microsoft/winget-pkgs/tree/master/doc/manifest/schema/1.2.0)?

Note: `<path>` is the name of the directory containing the manifest you're submitting.

-----

###### Microsoft Reviewers: [Open in CodeFlow](https://portal.fabricbot.ms/api/codeflow?pullrequest=https://github.com/microsoft/winget-pkgs/pull/87920)

### CODE REVIEW

**Comment by itmunky** [comment]
@microsoft-github-policy-service agree

File: manifests/w/WatchGuard/MobileVPNwithSSLclient/12.7.2/WatchGuard.MobileVPNwithSSLclient.locale.en-US.yaml [hunk 1]
```diff
@@ -0,0 +1,14 @@
+# Created using wingetcreate 1.1.2.0
+# yaml-language-server: $schema=https://aka.ms/winget-manifest.defaultLocale.1.2.0.schema.json
+
+PackageIdentifier: WatchGuard.MobileVPNwithSSLclient
+PackageVersion: 12.7.2
+PackageLocale: en-US
+Publisher: WatchGuard
+PackageName: WatchGuard Mobile VPN with SSL Client
```
Comment (ItzLevvie) [inline manifests/w/WatchGuard/MobileVPNwithSSLclient/12.7.2/WatchGuard.MobileVPNwithSSLclient.locale.en-US.yaml:8]:
```suggestion
PackageName: WatchGuard Mobile VPN with SSL client
```
This must match Control Panel.

**Review by ItzLevvie — CHANGES_REQUESTED** [review]

**Comment by ItzLevvie** [comment]
@msftbot reset feedback

**Comment by ItzLevvie** [comment]
@msftbot Blocking-Issue

**Pull request closed without merging**